packages/ui/src/working-groups/types/Reward.ts

```diff
@@ -0,0 +1,6 @@
+import BN from 'bn.js'
+
+export interface Reward {
+  value: BN
+  interval: number
```

Comment:
We might forget what was what ;)

```suggestion
  payout: BN
  blockInterval: number
```

Reply:
Payout - because this would be a payout done to the worker (this is also used in schema / node code).